secant/Wrappers/WrappedCaptureDevice.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  WrappedCaptureDevice.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 17.05.2022.
+//
+
+import Foundation
+import AVFoundation
+
+struct WrappedCaptureDevice {
+    let torch: (Bool) -> Void
+}
+
+extension WrappedCaptureDevice {
+    static let real = WrappedCaptureDevice(
+        torch: { isTorchOn in
+            if let videoCaptureDevice = AVCaptureDevice.default(for: .video) {
+                do {
+                    try videoCaptureDevice.lockForConfiguration()
+                    videoCaptureDevice.torchMode = isTorchOn ? .on : .off
+                    videoCaptureDevice.unlockForConfiguration()
+                } catch { }
```

Comment:
can you add a TODO to handle this error?

Reply:
improved the code and created todo